### ISSUE TITLE
Server API session creation response is not compatible with JS SDK

### ISSUE THREAD

**Issue by powli** [Ingenico-ePayments/connect-sdk-client-js#7]
When I create a session through the PHP Server SDK I get a response similar to this:
```
{
    "assetUrl": "https:\/\/payment.pay1.preprod.secured-by-ingenico.com\/",
    "clientApiUrl": "https:\/\/ams1.preprod.api-ingenico.com\/client",
    "clientSessionId": "<session-id>",
    "customerId": "<client-id>",
    "region": "EU"
}
``` 
However this can not directly be used to create a JavaScript SDK Session, since it expects the following properties (see [here](https://github.com/Ingenico-ePayments/connect-sdk-client-js/blob/master/src/C2SCommunicatorConfiguration.js#L119-L134)):

- clientSessionID
- apiBaseUrl
- assetsBaseUrl

Do I really have to manually map the session properties?

-----

Also the `clientApiUrl` is not usable directly, since the [tester](https://github.com/Ingenico-ePayments/connect-sdk-client-js/blob/master/src/C2SCommunicatorConfiguration.js#L152-L157) expects the url to end with `client/v1` instead of just `client`, which is actually returned from the `getSession` endpoint.

**Comment by rob-spoor** [comment]
The next release will allow the response to be used as session details. The `C2SCommunicatorConfiguration` class will make sure that both the old session details structure (with `clientSessionID`, `customerId`, `region` and `environment`) and the create session response will work. Just make sure to:
* not provide both `clientSessionId` and `clientSessionID`.
* not provide both `assetUrl` and `assetsBaseUrl`.
* not provide both `clientApiUrl` and `apiBaseUrl`.
* either provide the `clientApiUrl` and `assetUrl` or provide the `region` and `environment` (the region will be ignored if the URLs are provided).

**Comment by ebomcke-ingenico** [comment]
Fixed in release 3.8.0.
